products/ionic.md

```diff
@@ -11,50 +11,66 @@ activeSupportColumn: true
 eolColumn: Maintenance
 releaseColumn: true
 releaseDateColumn: true
+extendedSupportColumn: true
 
 auto:
 -   git: https://github.com/ionic-team/ionic-framework.git
 
+# support(R) = releaseDate(R+1)
 releases:
--   releaseCycle: "6"
-    releaseDate: 2021-12-08
+-   releaseCycle: "7"
+    releaseDate: 2023-03-29
     support: true
     eol: false
+    extendedSupport: false
```

Comment:
There is extended support, but end date is not defined yet. So should be `true` ?